.github/workflows/publish.yml

```diff
@@ -0,0 +1,34 @@
+name: NPM Publish
+
+on:
+  release:
+    types: [created]
+  workflow_dispatch:
+
+env:
+  NODE_VERSION: 16
+  NODE_AUTH_TOKEN: ${{ secrets.NPM_PUBLISH_TOKEN }}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+          registry-url: https://registry.npmjs.org/
+      - name: Install Python setuptools
+        # Reasoning for this in `ci.yml`
+        run: |
+          python3 -m venv CI_venv
+          source CI_venv/bin/activate
+          python3 -m pip install setuptools
+      - name: Install Dependencies
+        run: |
+          source CI_venv/bin/activate
+          npm ci
```

Comment:
Give this a try:

```suggestion
          npm config set python "$(which python)"
          npm ci
```

Reply:
(stealth-edited `python` to `python3` there, so don't miss it :))

Reply:
Great find above, I'll apply these changes